src/components/SettlementButton.tsx

```diff
@@ -152,33 +174,50 @@ function SettlementButton({
     const isPaidGroupPolicy = ReportUtils.isPaidGroupPolicyExpenseChat(chatReport);
     const shouldShowPaywithExpensifyOption = !isPaidGroupPolicy || (!shouldHidePaymentOptions && ReportUtils.isPayer(session, iouReport as OnyxEntry<Report>));
     const shouldShowPayElsewhereOption = !isPaidGroupPolicy || policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL;
+    const isExpenseReport = ReportUtils.isExpenseReport(iouReport ?? null);
+    const paymentCardList = fundList ?? {};
+    const hasBankAccount = isExpenseReport && reimbursementAccount?.achData?.state !== CONST.BANK_ACCOUNT.STATE.OPEN && !isExpenseReport && bankAccountList !== null;
+    const hasPaymentMethod = bankAccountList !== null && PaymentUtils.hasExpensifyPaymentMethod(paymentCardList, bankAccountList, true) && shouldShowPaywithExpensifyOption;
+
     const paymentButtonOptions = useMemo(() => {
-        const buttonOptions = [];
-        const isExpenseReport = ReportUtils.isExpenseReport(iouReport);
         const paymentMethods = {
-            [CONST.IOU.PAYMENT_TYPE.EXPENSIFY]: {
-                text: translate('iou.settleExpensify', {formattedAmount}),
-                icon: Expensicons.Wallet,
-                value: CONST.IOU.PAYMENT_TYPE.EXPENSIFY,
-            },
             [CONST.IOU.PAYMENT_TYPE.VBBA]: {
                 text: translate('iou.settleExpensify', {formattedAmount}),
                 icon: Expensicons.Wallet,
                 value: CONST.IOU.PAYMENT_TYPE.VBBA,
             },
+            [CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT]: {
+                text: translate('iou.settlePersonalBank', {formattedAmount}),
+                icon: Expensicons.Bank,
+                value: CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT,
+            },
```

Comment:
Shouldn't we include this only if `shouldShowPersonalBankAccountOption` is `true`?

Reply:
Is this resolved?